src/styles/clock.scss

```
@@ -0,0 +1,83 @@
+.clock {
+  position: relative;
+  width: 600px;
+  height: 600px;
+
+  margin: 150px auto;
+
+  box-sizing: border-box;
+  border-radius: 50%;
+  border: 9px solid #424242;
```

```suggestion
  border: 9px solid #424242;
```
цвет можно записать в переменную